common/index.ftd

```diff
@@ -0,0 +1,504 @@
+-- import: fastn/processors as pr
+
+-- string site-name: Fastn
+
+-- boolean $tc-pop-up-status: false
+
+-- boolean show-footer: true
+
+-- ftd.image-src site-logo: $assets.files.assets.logo.svg
+
+-- integer logo-width: 180
+
+-- integer $current-slide: 1
+
+-- integer logo-height: 44
+
+-- string site-url: index.html
+
+-- boolean login-button: false
+
+-- integer max-width: 1320
+
+-- pr.sitemap-data sitemap:
+$processor$: pr.sitemap
+
+-- pr.sitemap-data footer-toc:
+$processor$: pr.full-sitemap
+
+-- fastn.app-ui apps:
+$processor$: pr.fastn-apps
+
+-- ftd.image-src fastn-logo:
+light: https://fastn.com/-/fastn.com/images/fastn.svg
+dark: https://fastn.com/-/fastn.com/images/fastn-dark.svg
+
+-- record social-media:
+ftd.image-src src:
+string link: /
+
+-- social-media list share:
+
+-- social-media list social-links:
+
+-- social-media:
+link: /
+src: $assets.files.assets.fb.png
+
+-- social-media: 
+link: /
+src: $assets.files.assets.figma.png
+
+-- social-media:
+link: /
+src: $assets.files.assets.google.png
+
+-- social-media: 
+link: /
+src: $assets.files.assets.chat.png
+
+-- social-media: 
+link: /
+src: $assets.files.assets.social.png
+
+-- end: social-links
+
+-- pr.toc-item list empty-toc:
+$processor$: pr.toc
+
+-- pr.toc-item list footer-links:
```

Comment:
@saurabh-lohiya check if this is not used, we need to remove this from all - aiglesoft, and wizard also.